src/components/Canvas/ImagesBlock.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div v-for="(relateImage, index) in relatedImages" :key="index">
+    <image-element
+      :imagePosition="calculatePositions(index)"
+      :isTop="true"
+      :isLeft="false"
```

Comment:
does not seem used

Reply:
Right, it's not currently used but necessary later. I already implement they but I can remove for this PR and reintroduce later.

Reply:
no need to remove if used later